app/views/tables.html

```diff
@@ -3,132 +3,129 @@
 <div ng-class="{'page-container-padded': renderSidebar}">
 <div class="page-container" ng-show="table">
 
-      <div class="row row-first">
-        <div class="col-sm-12">
-          <a class="btn btn-default menu-toggle" ng-click="toggleSidebar()"><i class="fa fa-angle-double-right"></i></a>
-        </div>
-        <div class="col-sm-6 col-xs-12">
-          <div class="panel panel-info">
-            <div class="panel-heading">
-              <p class="announcement-heading"><b>
-                <ng-pluralize ng-init="translationKey='TABLE.TITLE'" count="nr_of_tables"
-                when="{
-                    '0': '{{ translationKey | translate }} ({})',
-                    '1': '{{ table_label }}',
-                    'other': '{{ table_label }}'}">
-              </ng-pluralize>
-              </b> <span ng-show="isParted" class="small">(partitioned)<span></p>
-              <p class="announcement-text">{{ 'TABLE.NAME' | translate }}</p>
-            </div>
-          </div>
-        </div>
-
-        <div class="col-sm-3 col-xs-6">
-          <div class="panel panel-info text-right {{table.health_panel_class}}">
-            <div class="panel-heading">
-              <p class="announcement-heading">{{table.health}}</p>
-              <p class="announcement-text">{{ 'TABLE.HEALTH' | translate }}</p>
-            </div>
-          </div>
-        </div>
-
-        <div class="col-sm-3 col-xs-6">
-          <div class="panel panel-info text-right">
-            <div class="panel-heading">
-              <p class="announcement-heading">{{table.replicas_configured}}</p>
-              <p class="announcement-text">{{ 'TABLE.CONFIG_REPLICAS' | translate }}</p>
-            </div>
-          </div>
-        </div>
+  <h1>{{ 'NAVIGATION.TABLE' | translate }}</h1>
+
+  <div class="table-detail cr-panel-block">
+    <div class="table-detail-row">
+      <div class="table-detail-cell">
+        <p class="table-detail-cell-title">
+          {{ 'TABLE.NAME' | translate }}
+        </p>
+        <p class="table-detail-cell-content">
+          <ng-pluralize ng-init="translationKey='TABLE.TITLE'" count="nr_of_tables"
+          when="{
+              '0': '{{ translationKey | translate }} ({})',
+              '1': '{{ table_label }}',
+              'other': '{{ table_label }}'}">
+        </ng-pluralize>
+        <span ng-show="isParted" class="small">(partitioned)<span>
```

Comment:
can you remove the class`small` or define it in common.less
we are removing all the dependencies to theme.less